gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -354,6 +426,19 @@ func main() {
 	assert.NoError(t, err)
 	require.NotNil(t, res)
 	assert.Equal(t, expected, string(res.DeliverTx.Data))
+
```

Comment:
Just curious: why aren't these tests parallel? Can you please make them?

Reply:
The integration tests set up an in-memory node, but I believe that the node listens on a TCP socket like tcp://127.0.0.1:62139 . Should many such nodes run in parallel?

Reply:
Ah, the TCP setup is not parallel friendly. Not a huge problem, let's keep it sequential for now